### ISSUE TITLE
AuthAD Plugin: Cannot log out when Single Sign-On is enabled

### ISSUE THREAD

**Issue by Lord-Dimwit-Flathead-the-Excessive** [splitbrain/dokuwiki#1735]
Site should present HTTP Response "401 Unauthorized" upon clicking LogOut.   The browser will then challenge for new credentials.

**Comment by splitbrain** [comment]
Not sure this makes sense. The idea of SSO is that the browser will NOT prompt for any credentials as it already knows who is logged in.

**Comment by Lord-Dimwit-Flathead-the-Excessive** [comment]
Yes and this is fine for users that only use a single Active Directory account.  For administrators, it makes perfect sense.  I won't be publishing using an administrative account and I certainly don't want my workstation logged into a privileged AD account just so I can admin the Dokuwiki site.

Additionally, when one is initially setting up ACLs, we are logging into many different AD accounts to test the access controls.

This is something I have done numerous times on Active Server based sites I have developed .  I have never attempted it using PHP but I'm sure I could.  I'm just not familiar enough with Dokuwiki's layout and plug-in best practices to be sure I put it in the right place.  I'd be happy to write it if you can point me to an appropriate module and function...

**Comment by splitbrain** [comment]
Have a look at https://www.dokuwiki.org/devel:auth_plugins The logOff() method should be a good start for implementing this.
